src/PerformanceSensitiveAnalyzers/Core/AbstractAllocationAnalyzer.cs

```diff
@@ -152,4 +150,65 @@ private static PerformanceSensitiveInfo CreatePerformanceSensitiveInfo(Attribute
             }
         }
     }
+
+    internal abstract class AbstractAllocationAnalyzer<TLanguageKindEnum>
+        : AbstractAllocationAnalyzer
+        where TLanguageKindEnum : struct
+    {
+        protected abstract ImmutableArray<TLanguageKindEnum> Expressions { get; }
+
+        protected override ImmutableArray<OperationKind> Operations => ImmutableArray<OperationKind>.Empty;
```

Comment:
💡 This needs to be `sealed` _or_ the implementation of `Initialize` needs to be updated to call `base.Initialize(context)`.